src/components/home/Sponsors.jsx

```diff
@@ -1,25 +1,85 @@
+"use client";
+
+import { motion } from "framer-motion";
+
+const animations = {
+  start: {
+    opacity: 0,
+    y: 30,
+  },
+  end: {
+    opacity: 1,
+    y: 0,
+  },
+};
+
 const Sponsors = () => {
   return (
     <div className="relative bg-witl-purple-300 w-[100%]">
       <div className="text-witl-purple-200 pl-[15%] md:pl-[10%] flex flex-col items-start leading-tight pt-4 md:pt-0">
-        <div className="lg:text-[8rem] mt-[10%] text-3xl md:text-6xl">OUR</div>
-        <div className="lg:text-10xl text-5xl md:text-7xl">
+        <motion.div
+          className="lg:text-[8rem] mt-[10%] text-3xl md:text-6xl"
+          variants={animations}
+          initial="start"
+          whileInView="end"
+          transition={{ delay: 0.2 }}
+        >
+          OUR
+        </motion.div>
+        {/* <div className="lg:text-[8rem] mt-[10%] text-3xl md:text-6xl">OUR</div> */}
+        <motion.div
+          className="lg:text-10xl text-5xl md:text-7xl"
+          variants={animations}
+          initial="start"
+          whileInView="end"
+          transition={{ delay: 0.2 }}
+        >
           &nbsp;&nbsp;SPONSORS
-        </div>
+        </motion.div>
+        {/* <div className="lg:text-10xl text-5xl md:text-7xl">
+          &nbsp;&nbsp;SPONSORS
+        </div> */}
       </div>
       <div className="absolute top-[15%] md:top-[20%] right-[0%] bg-witl-purple-400 text-witl-white-100 font-libre-caslon-text font-bold text-[49.37px] w-[55%]">
```

Comment:
```suggestion
      <div className="absolute top-[15%] md:top-[20%] right-[0%] bg-witl-purple-300 text-witl-white-100 font-libre-caslon-text font-bold text-[49.37px] w-[55%]">
```
you're running into a problem where the blueprint test prep background isn't being animated into view because the outer div is what's bg-witl-purple-400 so change it to 300 and set the inner div to have the bg-witl-purple-300

Reply:
https://github.com/user-attachments/assets/280b100c-d070-4a3e-ad6c-4a87203455e1

fixed animations